### DISCUSSION TITLE
♿ Les labels de la repartition des cotisations contiennent une information uniquement via la couleur

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2663] [Idées]
![Screenshot_2020-12-09 Auto-entrepreneur calculez rapidement votre revenu net à partir du CA et vice-versa](https://user-images.githubusercontent.com/1775934/101633081-11b2d400-3a27-11eb-9dc5-93fa56c022a7.png)

On peut imaginer ajouter des numéro dans les barres référencés dans la légende.

**Comment by mquandalle** [comment]
Des informations au survol de la barre c'est acceptable pour l'accessibilité ?
J'imagine que ça ne fonctionne pas pour les "lecteurs en mode texte" mais dans ce cas là, la barre tout entière est inutile et les infos sont dans la légende.

**Comment by johangirod** [comment]
> Des informations au survol de la barre c'est acceptable pour l'accessibilité ?

Ca pose problème pour le mobile, et ça dégrade l'experience pour les personnes qui ne voient pas les couleurs. Je pense que nombre + couleur c'est un bon combo.
